src/lib/utils/highlighter.ts

```diff
@@ -22,10 +23,10 @@ const supportedLanguages = unique([
 
 let highlighter: Highlighter | undefined;
 
-export async function loadHighlighter() {
+export async function loadHighlighter(theme: Theme) {
     if (highlighter) return;
     highlighter = await shiki.getHighlighter({
-        theme: "light-plus",
+        theme: theme || "light-plus",
```

Comment:
I'd rather specify this as the default in the option declaration - then it can be picked up and shown in the typedoc.json schema file.